pages/callbacks/__init__.py

```diff
@@ -1 +1 @@
-# empty file to make this a module
```

Comment:
files should be taken off of PR as nothing has been changed

Reply:
it's going to end up in some PR somewhere because it's based on the Black python formatter. Otherwise we'd have to ignore this file.